app/assets/javascripts/system/admin/instance/users.js

```diff
@@ -0,0 +1,10 @@
+//= require helpers/table_row_form_helpers
+
+(function($, TABLE_ROW_FORM_HELPERS) {
+  'use strict';
+
+  var DOCUMENT_SELECTOR = '.system-admin-instance-users.index ';
+  var BUTTON_SELECTOR = 'tr.instance-user #update';
+
+  TABLE_ROW_FORM_HELPERS.initialize(DOCUMENT_SELECTOR + BUTTON_SELECTOR);
```

Comment:
Think it would be better to rename `initialize` or the helper to make it more obvious ?  I read this line still don't know what's the method is doing.  What do you think ?